File: 1Trimestre/Ejercicio1.java
package ejerciciosJava1Trimestre;
import java.util.Scanner;
public class Ejercicio_1 {
	public static void main(String[] args) {
		
		System.out.println("Introduzca su fecha de nacimiento en formato dd/mm/aa");
		
		int diaNacimiento,mesNacimiento,anoNacimiento,diaActual,mesActual,anoActual;
			
		@SuppressWarnings("resource")
		Scanner scanner = new Scanner(System.in);
			diaNacimiento = scanner.nextInt();
			mesNacimiento = scanner.nextInt();
			anoNacimiento = scanner.nextInt();
			
		System.out.println("Introduzca la fecha actual en formato dd/mm/aa");	
			diaActual = scanner.nextInt();
			mesActual = scanner.nextInt();
			anoActual = scanner.nextInt();
			
		if (anoActual-anoNacimiento>65) {
			System.out.println("Usted puede jubilar");
		}else{
			if (anoActual-anoNacimiento==65) {
				if (mesActual>mesNacimiento) {	
					System.out.println("Usted puede jubilar");				
					
				}else {
					if (mesActual==mesNacimiento) {
						if(diaActual>=diaNacimiento) {
							System.out.println("Usted puede jubilarse");
						}
					}
				}	
			}
		}
		if(anoActual-anoNacimiento<65) {
			System.out.println("Usted no puede jubilarse");
		}else {
			if(anoActual-anoNacimiento==65) {
				if(mesActual<mesNacimiento) {
					System.out.println("Usted no puede jubilarse");
				}else {
					if(mesActual==mesNacimiento) {
						if(diaActual<diaNacimiento) {
							System.out.println("Usted no puede jubilarse");
						}
					}
				}
			}
		}
	}
}

File: 1TrimestrePython/Ejercicio1.py
dia_nacimiento=0
mes_nacimiento=0
anno_nacimiento=0

dia_actual=0
mes_actual=0
anno_actual=0

dia_nacimiento=int (input("Introduzca su día de nacimiento"))
mes_nacimiento=int (input("Introduzca su mes de nacimiento"))
año_nacimiento=int (input("Introduzca su año de nacimiento"))

dia_actual=int (input("Introduzca el día actual"))
mes_actual=int (input("Introduzca el mes actual"))
año_actual=int (input("Introduzca el año actual"))

if anno_actual-anno_nacimiento>65:
    print("Usted puede jubilarse")

else:
    if anno_actual-anno_nacimiento==65:
        if mes_actual>mes_nacimiento:
            print("Usted se puede jubilar")
        else:
            if mes_actual>=mes_nacimiento:
                if dia_actual>=dia_nacimiento:
                    print("Usted se puede jubilar")


if anno_actual-anno_nacimiento<65:
    print("Usted no se puede jubilar")
else:
    if anno_actual-anno_nacimiento==65:
        if mes_actual<mes_nacimiento:
            print("Usted no puede jubilarse")
        else:
            if mes_actual==mes_nacimiento:
                if dia_actual<dia_nacimiento:
                    print("Usted no puede jubilarse")
   
